src/main/java/org/venice/beachfront/bfapi/services/JobService.java

```diff
@@ -106,12 +115,12 @@ public Job createJob(String jobName, String creatorUserId, String sceneId, Strin
 		Scene scene = null;
 		try {
 			scene = sceneService.getScene(sceneId, planetApiKey, true);
-		} catch (Exception exception) {
+		} catch (UserException exception) {
```

Comment:
Why are we erasing and re-emitting UserExceptions? The method already `throws UserException` so why not just... not catch these?

Reply:
Good point. This is a relic of before the Scene Service switched over to user exceptions.  I will remove the unnecessary try/catch block.